dll/apisets/api-ms-win-crt-runtime-l1-1-0.spec

```diff
@@ -52,7 +52,7 @@
 @ stub _get_thread_local_invalid_parameter_handler
 @ stub _get_wide_winmain_command_line
 @ stdcall -version=0x600+ _get_wpgmptr() msvcrt._get_wpgmptr
-@ stdcall -version=0x502 -arch=i386,x86_64 _getdllprocaddr() msvcrt._getdllprocaddr
+@ stdcall -arch=i386,x86_64 _getdllprocaddr() msvcrt._getdllprocaddr
```

Comment:
This apiset is not gonna load on post 0x502 x64 builds.

Reply:
I checked on msvcrt spec file it should be ok now